email_campaigns/email_to_mentors.txt

```diff
@@ -0,0 +1,47 @@
+Subject: You’re mentoring for the mentored sprint for diverse beginners at PyCon US!
+
+{Full name},
+
+Thank you for signing up to mentor for the {project} project at the mentored sprint for diverse beginners at PyCon US.
+We’re pleased to welcome you to the sprint.
+
+The mentored sprint will take place on **Saturday, May 4th 2:30 PM - 6:30 PM. Room 25C**
+
+Some useful information:
+
+- The mentored sprint is organized as part of PyCon US. You will need to be registered
+  for PyCon in order to attend the mentored sprint. If you have not registered, please do
+  so as soon as possible, because PyCon is a sell out year after year.
+ 
+- We're excited that numerous projects and mentors have signed up for the sprint! Each
+  project will have the opportunity to pitch for their own project to the group. Please
+  be prepared with a **3 minute pitch**. Feel free to bring a slide (or two) with more
```

Comment:
For the slides, will the project need to send them to us in advance? If so, in what format? Otherwise we'll need to fiddle with switching A/V over from computer to computer. Something to think about.

Reply:
Yes, shall we request these in pdf? Or Google slides one day before at let's say 5pm (Eastern time, this si Cleveland time right?) so that we can serve them from one of our computers?

Reply:
Added mention to send us Google slides a day before.